packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx

```diff
@@ -62,26 +50,36 @@ export const ArrowIndicatorCell = ({
       >
         {arrowUp !== null ? (
           <>
-            <SmallTableCell sx={{ color: valueTextColor }}>
-              {value}
-            </SmallTableCell>
-            <AngledArrowIcon
+            <SmallTableCell
               sx={{
-                marginLeft: spacing(0.5),
-                marginRight: spacing(-0.25),
-                color: tooRisky
-                  ? palette.error.main
-                  : arrowUp
-                  ? colorOnUp
-                  : colorOnDown,
-                width: spacing(1.5),
-                transform: arrowUp ? 'rotate(180deg)' : 'rotate(270deg)',
+                color: isHealthFactor
+                  ? cell?.row?.original?.textColor
```

Comment:
Can we remove the `isHealthFactor` boolean and just do `cell?.row?.original?.textColor || valueTextColor` something like that?